examples/seq_to_lang_sample.py

```diff
@@ -0,0 +1,16 @@
+import os
+import sys
+from seqtolang import Detector
+
+detector = Detector()
+
+if (os.environ.get('SEQTOLANG_TEXT') is None):
```

Comment:
Not sure it is intuitive enough for the average user of tis lib.
I mean, ideally, regardless of the Docker thing, the average use case for would-be cloning this repo and then running examples, which won't work in this state. Why we need an envvar here?